theme/templates/blog.html

```diff
@@ -7,7 +7,7 @@
     <div class="flex flex-row flex-wrap -mx-2">
       {% for article in articles_page.object_list %}
         <div class="w-full sm:w-full md:w-1/2 mb-4 px-2">
-          <div class="max-w-4xl px-10 my-4 py-6 bg-white rounded-lg shadow-md">
+          <div class="max-w-4xl flex flex-col h-full px-10 my-4 py-6 bg-white rounded-lg shadow-md">
```

Comment:
Just being curious about the layout, I've tried comparing the change before (like the code below) and after, 
but I found previous version it works fine on my screen. 
```suggestion
          <div class="max-w-4xl h-full px-10 my-4 py-6 bg-white rounded-lg shadow-md">
```

<details>
<summary>Screenshots</summary>

![image](https://github.com/sciwork/swportal/assets/34214497/1730928f-754e-40db-83be-61c39df54830)

![image](https://github.com/sciwork/swportal/assets/34214497/94bf03a6-5edc-42b7-b378-485b020119bb)

</details>

Reply:
Oh, I've tried to shorten the blog article, which can see the difference now.

<details>
<summary>Screenshots</summary>

![image](https://github.com/sciwork/swportal/assets/34214497/e31394e7-f4e6-401f-bbe7-fa2e7a935fee)

</details>